src/components/ArticleCards.jsx

```diff
@@ -0,0 +1,33 @@
+import { useEffect, useState } from "react"
+import { getArticles } from "./api"
+import "../styling/ArticleCards.css";
+
+
+export const ArticleCards = () => {
+
+const [articles, setArticles] = useState([])
+
```

Comment:
it would be nice to have some loading state. So if the website was taking a long time to load the user could see a <p> tag to know it was loading rather than there being an error